### PR TITLE
sklearn.preprocessing.normalize -> from sklearn.preprocessing import …

### PR DESCRIPTION
**Pull request by crystina-z**
For issue #1112

### CODE REVIEW

**Review by lintool — APPROVED** [review]

**Comment by lintool** [comment]
hey @crystina-z any idea why this wasn't caught by our unit tests? Is it a sklearn version issue?

**Comment by crystina-z** [comment]
@lintool not really a sklearn version issue I think. the line is only triggered when the `self.l2_norm` is set to True, which only happens when using SentenceTransformer - I assume we don't have a test case for that?

where the SentenceTransformer calls this function - https://github.com/castorini/pyserini/blob/master/pyserini/encode/__main__.py#L33
